tests/input/UnusedVariables.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
I've created a new test file to help us test future unused variables in functions scopes :+1: